addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -194,6 +194,14 @@ def _l10n_in_edi_ewaybill_cancel_invoice(self, invoices):
             res[invoices] = inv_res
         return res
 
+    def _l10n_in_edi_ewaybill_handle_zero_distance_alert_if_present(self, invoices, response):
+        if invoices.l10n_in_distance == 0 and (alert := response.get("data", {}).get('alert')):
+            pattern = r", Distance between these two pincodes is \d+, "
+            if re.fullmatch(pattern, alert) and (distance := int(re.search(r'\d+', alert).group())) > 0:
+                invoices.l10n_in_distance = distance
+            else:
+                _logger.info("Alert Message Pattern did not matched from Government GST API: %s", alert)
```

Comment:
What happens if we have two alerts?  One about the distance and one for something else?  It seems that this log might just bring confusion then.

Reply:
That is the reason for logger at least we can know what other if there are any other type of alert we are receiving. 
As of now we only know about this as it is from production test by @jva-odoo .

Reply:
If you want we can remove the _logger anyways we want the pattern to be matched

Reply:
Then it should be: 
```suggestion
                _logger.info("Alert Message from Government GST API: %s", alert)
```

Reply:
The simplest might be indeed to just remove it.